api_tests/guids/views/test_guid_detail.py

```diff
@@ -1,53 +1,69 @@
-from nose.tools import *  # flake8: noqa
-from api.base.settings.defaults import API_BASE
 import pytest
 
 from addons.osfstorage.models import OsfStorageFile
-from website.settings import API_DOMAIN
-
-from tests.base import ApiTestCase
-from osf_tests.factories import (AuthUserFactory, ProjectFactory, RegistrationFactory,
-                                 CommentFactory, CollectionFactory, PrivateLinkFactory)
 from addons.wiki.tests.factories import NodeWikiFactory
+from api.base.settings.defaults import API_BASE
+from osf_tests.factories import (
+    AuthUserFactory,
+    ProjectFactory,
+    RegistrationFactory,
+    CommentFactory,
+    CollectionFactory,
+    PrivateLinkFactory,
+)
+from website.settings import API_DOMAIN
 
-
-class TestGuidDetail(ApiTestCase):
-
-    def setUp(self):
-        super(TestGuidDetail, self).setUp()
-        self.user = AuthUserFactory()
-
-    def _add_private_link(self, project, anonymous=False):
-        view_only_link = PrivateLinkFactory(anonymous=anonymous)
-        view_only_link.nodes.add(project)
-        view_only_link.save()
-        return view_only_link
-
-    def test_redirect_to_node_view(self):
-        project = ProjectFactory()
-        url = '/{}guids/{}/'.format(API_BASE, project._id)
-        res = self.app.get(url, auth=self.user.auth)
+@pytest.mark.django_db
+class TestGuidDetail:
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def project(self):
+        return ProjectFactory()
+
+    @pytest.fixture()
+    def add_private_link(self, project):
+        def private_link(anonymous=False):
+            view_only_link = PrivateLinkFactory(anonymous=anonymous)
+            view_only_link.nodes.add(project)
+            view_only_link.save()
+            return view_only_link
+        return private_link
+
+    @pytest.fixture()
```

Comment:
Is anything gained by this being a fixture? It looks like it could either be a simple function (or just inline since it isn't that complicated)

```
def format_url(id):
    return '/{}guids/{}/'.format(API_BASE, id)
```